src/Tasks/AssemblyDependency/ReferenceTable.cs

```diff
@@ -971,7 +971,8 @@ private void FindSatellites(
                     // Is there a candidate satellite in that folder?
                     string cultureName = Path.GetFileName(subDirectory);
 
-                    if (CultureInfoCache.IsValidCultureString(cultureName))
+                    // Custom or unknown cultures can be met as well
+                    if (ChangeWaves.AreFeaturesEnabled(ChangeWaves.Wave17_14) || CultureInfoCache.IsValidCultureString(cultureName))
```

Comment:
So after the change wave is rotated out, we can discard this if altogether?

Reply:
Exactly.
Whenever removing stale changewaves, it's expected to remove conditionality that becames peremanent (including  removing code that becames unreachable - not a case here though)